client/reader/sidebar/index.jsx

```diff
@@ -128,111 +127,105 @@ export class ReaderSidebar extends React.Component {
+								link={ '/read/conversations' }
+							/>
+
+							<ReaderSidebarTeams teams={ teams } path={ this.props.path } />
+
+							{ isAutomatticTeamMember( teams ) && (
+								<SidebarItem
 									className={ ReaderSidebarHelper.itemLinkClass(
 										'/read/conversations/a8c',
 										this.props.path,
 										{
 											'sidebar-streams__conversations': true,
 										}
 									) }
-								>
-									<a
-										href="/read/conversations/a8c"
-										onClick={ this.handleReaderSidebarA8cConversationsClicked }
-									>
-										<svg
-											className={ 'gridicon gridicon-automattic-conversations' }
-											width="24"
-											height="24"
-											xmlns="http://www.w3.org/2000/svg"
-											viewBox="0 0 24 24"
-										>
-											<path d="M12.2 7.1c.5.3.6 1 .3 1.4L10 12.4c-.3.5-1 .7-1.4.3-.6-.3-.8-1-.4-1.5l2.5-3.9c.3-.4 1-.5 1.5-.2zM17.3 21.2h2.8c1 0 1.9-.8 1.9-1.9v-4.7c0-1-.8-1.9-1.9-1.9h-7.6c-1 .1-1.7.9-1.7 1.9v4.7c0 1 .8 1.8 1.7 1.9h2V24l2.8-2.8z" />
-											<path d="M8.8 15.2c-2.7-.7-4.1-2.9-4.1-5.2 0-5.8 5.8-5.7 5.8-5.7 5.8 0 5.8 5.7 5.8 5.7 0 .3 0 .6-.1.8H19v-.7C19 1.6 10.4 2 10.4 2c-8.6 0-8.5 8.1-8.5 8.1 0 3.5 2.7 6.8 6.9 7.5v-2.4z" />
-										</svg>
-										<span className="menu-link-text">A8C Conversations</span>
-									</a>
-								</li>
+									label="A8C Conversations"
+									onNavigate={ this.handleReaderSidebarA8cConversationsClicked }
+									link={ '/read/conversations/a8c' }
+									customIcon={ this.renderA8CConversationsIcon() }
+								/>
 							) }
 
 							{ isDiscoverEnabled() ? (
-								<li
+								<SidebarItem
 									className={ ReaderSidebarHelper.itemLinkClass( '/discover', this.props.path, {
 										'sidebar-streams__discover': true,
 									} ) }
-								>
-									<a href="/discover" onClick={ this.handleReaderSidebarDiscoverClicked }>
-										<Gridicon icon="my-sites" />
-										<span className="menu-link-text">{ this.props.translate( 'Discover' ) }</span>
-									</a>
-								</li>
+									label={ this.props.translate( 'Discover' ) }
+									onNavigate={ this.handleReaderSidebarDiscoverClicked }
+									icon="my-sites"
+									link={ '/discover' }
+								/>
 							) : null }
 
-							<li
+							<SidebarItem
+								label={ this.props.translate( 'Search' ) }
+								onNavigate={ this.handleReaderSidebarSearchClicked }
+								materialIcon="search"
+								link={ '/read/search' }
 								className={ ReaderSidebarHelper.itemLinkClass( '/read/search', this.props.path, {
 									'sidebar-streams__search': true,
 								} ) }
-							>
-								<a href="/read/search" onClick={ this.handleReaderSidebarSearchClicked }>
-									<MaterialIcon icon="search" />
-									<span className="menu-link-text">{ this.props.translate( 'Search' ) }</span>
-								</a>
-							</li>
+							/>
 
-							<li
+							<SidebarItem
+								label={ this.props.translate( 'My likes' ) }
```

Comment:
Minor, but it's currently "My Likes" and that feels like the more consistent option (for example, it's currently "Followed **S**ites".

```suggestion
								label={ this.props.translate( 'My Likes' ) }
```

<img width="205" alt="Screenshot 2019-08-28 at 08 58 20" src="https://user-images.githubusercontent.com/43215253/63836726-00e56880-c972-11e9-92b9-88120154eee5.png">